Sources/Routing/Parameters.swift

```diff
@@ -0,0 +1,26 @@
+import Node
+
+public final class ParametersContext: Context {
+    internal static let shared = ParametersContext()
+    fileprivate init() {}
+}
+
+public let parametersContext = ParametersContext.shared
+
+public struct Parameters: StructuredDataWrapper {
+    public static var defaultContext: Context? = parametersContext
+    public var wrapped: StructuredData
+    public let context: Context
+    
+    public init(_ wrapped: StructuredData, in context: Context? = defaultContext) {
+        self.wrapped = wrapped
+        self.context = context ?? parametersContext
+    }
+}
+
+extension Parameters {
+    public func get<P: Parameterizable>(_ p: P.Type = P.self) throws -> P {
```

Comment:
Have we verified that we don't get ambiguous warnings outside of this module when using this if an object conforms to Parameterizable _and_ NodeInitializable